bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableOptionsFactory.java

```diff
@@ -15,9 +15,6 @@
  */
 package com.google.cloud.bigtable.hbase;
 
-import static com.google.cloud.bigtable.config.BigtableOptions.BIGTABLE_ADMIN_HOST_DEFAULT;
```

Comment:
for my understanding - why are these getting deleted? are they getting set elsewhere already for default hosts?

Reply:
bigtable-client-core already has defaults, we were double setting the defaults here. This leads to duplicating the logic in bigtable-client-core and bigtable-hbase. WIth directpath, this logic is more complex than a simple const. So this PR only overrides the default if it's actually specified.

Reply:
SGTM. Can we add a test asserting the overrides are getting set properly?

Reply:
done